src/main/java/synapseawsconsolelogin/Auth.java

```diff
@@ -424,4 +449,18 @@ private String getSSMParameter(String name) {
 		}
 	}
 
+	public String initAppVersion() {
+		String version;
+		if (! getProperty(GIT_TAG_PROPERTY_KEY, false).isEmpty()) {
+			version = getProperty(GIT_TAG_PROPERTY_KEY);
+		} else {
+			version = String.format("{}-{}", getProperty(GIT_COMMIT_TIME_KEY), getProperty(GIT_COMMIT_ID_ABBREV_KEY));
```

Comment:
i don't think it's appropriate to use this class's getProperty method.  The purpose of that method is to allow override properties from java env, env vars, ssm, etc..   Git version info should never be overriden.